frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/InferenceServiceServingRuntimeSection.tsx

```diff
@@ -56,6 +52,7 @@ const InferenceServiceServingRuntimeSection: React.FC<
           key: servingRuntime.metadata.name,
           label: getDisplayNameFromK8sResource(servingRuntime),
         }))}
+        isSkeleton={!loaded && data.project !== ''}
```

Comment:
This removes the `&& !currentServingRuntime` case from the existing logic here, was that intentional?

Reply:
Hi, yes. If `currentServingRuntime` is present, it will always fall into
```
return (
      <FormGroup label="Model server">
        <Text>{getDisplayNameFromK8sResource(currentServingRuntime)}</Text>
      </FormGroup>
    );
```
above.
So the logic of checking `!currentServingRuntime` is not needed here anymore.

Reply:
Ah! Good call, ok.